### PR TITLE
Restapi auto redis/resource manager

### PR DESCRIPTION
**Pull request by kpajdzik**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: lib/services/rediscachemanagement/package.json [hunk 1]
```diff
@@ -1,37 +1,25 @@
 {
   "name": "azure-arm-rediscache",
   "author": "Microsoft Corporation",
-  "contributors": [
-    "Thaniyulla Parambath, Alfan <alfantp@microsoft.com>",
-    "Dacosta, Carlyle <cadaco@microsoft.com>",
-    "Zavery, Amar <amzavery@microsoft.com>"
-  ],
+  "description": "RedisManagementClient Library with typescript type definitions for node",
   "version": "2.1.0-preview",
```
Comment (daschult) [inline lib/services/rediscachemanagement/package.json]: Need to bump the version (I'm guessing 3.0.0).

File: lib/services/rediscachemanagement/README.md [hunk 2]
```diff
@@ -1,113 +0,0 @@
-# Microsoft Azure SDK for Node.js - Redis
```
Comment (daschult) [inline lib/services/rediscachemanagement/README.md]: You probably don't want to delete the README.md file.

Comment (kpajdzik) [reply lib/services/rediscachemanagement/README.md]: I removed the directory and regenerated. Is README written manually?

File: lib/services/rediscachemanagement/package.json (hunk 1 above)
Comment (kpajdzik) [reply lib/services/rediscachemanagement/package.json]: Yes, I'm waiting on publishing of autorest.nodejs so I can use the official package to regenerate.

File: lib/services/rediscachemanagement/README.md (hunk 2 above)
Comment (daschult) [reply lib/services/rediscachemanagement/README.md]: In this package it seems that it is.

**Pull request merged**